hack/e2e/run.sh

```diff
@@ -231,15 +236,16 @@ else
 
     set -x
     set +e
+    #TODO: revert #246 once a stable release including windows nodeSelector change is available; set to v1.27.0-alpha.3 for testing
```

Comment:
Should we have a Github issue to track this to-do?

Reply:
Not necessary, this will be fixed before we merge it. This PR will not be merged until K8s 1.27 is released.